project-ideas_2023/README.md

```diff
@@ -0,0 +1,272 @@
+| [Create a new plugin](https://github.com/RADAR-base/radar-commons-android#creating-a-plugin) for Google Sleep API.                | Retrieve any detected sleep start/end times and send data to RADAR-base server. The APIs require min Android 10. A minSdkVersion and compileSdkVersion of 29 or higher.                                                           |
+| Create schemas to send data from Google Activity Recognition API.                                                                 | Refer to [RADAR-Schemas](https://github.com/RADAR-base/RADAR-Schemas), create corresponding schemas and specification to send data from Google Activity Recognition API to RADAR-base platform. |
+| [Create a new plugin](https://github.com/RADAR-base/radar-commons-android#creating-a-plugin) for Google Activity Recognition API. | Retrieve any activity updates and detected activities and send data to RADAR-base server.                                                                                                       |
+
+**Required Skills:** Kotlin, Java
+
+**Difficulty:** Hard
+
+**Expected Size:** 350-hour (Full time)
+
+**Mentors**: [blootvoets](https://github.com/blootsvoets), [peyman-mohtashami](https://github.com/peyman-mohtashami)    ****TBD***
+
+---
+
+### 7. Real Time Data Analysis Model Builder
+
+**Overview**: The [model-builder](https://github.com/RADAR-base/model-builder) is a real time machine learning training, prediction and invocation module. It provides flexible infrastructure to add custom machine learning and deep learning models using any of the deep learning platforms. The model-builder reads data from the data streams and used them to predict in real time. The prediction then can be sent back to the streams.
+
+**Current Status**: We ran the pilot study using the model-builder and was able to predict the outcomes in real time. The platform is still in development stage and would require some major refactoring to make it more flexible and easily deployable.
+
+**Source Code**: https://github.com/RADAR-base/model-builder
+
+**Goals:**
+
+| Milestones                                         | Description                                                                                                                                             |
+|----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Kubernetes Integration         | Intergrating model-builder stack with the [Radar-Kubernetes](https://github.com/radar-base/radar-kubernetes/)                                                           |
```

Comment:
@Hsankesara  i think you can skip the kubernetes one in this as it is quite different from the other tasks and would require good knowledge of k8s, helm, helmfile and the r-base stack. It is also not mentioned in the skills. We can do the integration with radar-k8s fairly quickly as we have done similar ones in the past.

Reply:
You can also add the image from [here](https://docs.google.com/document/d/1vpybkkWRfYF3BLT-Uvemzrjd30knLfT1NMnkOnbbPjI/edit#heading=h.39me92fsq0kj) if you want (not required).